views/cloud-security-principles.erb

```diff
@@ -148,7 +148,7 @@
 				<li>GOV.UK PaaS team onboarding</li>
 			</ul>
 
-			<p>GDS follows government and Cabinet Office personnel security processes. All GDS personnel must have at least a security clearance of Security Checked (SC).</p>
+			<p>GDS follows government and Cabinet Office personnel security processes. All GDS personnel must pass a Baseline Personnel Security Standard (BPSS) check. Personnel with access to production systems must have at least a security clearance of Security Checked (SC).</p>
```

Comment:
```suggestion
			<p>GDS follows government and Cabinet Office personnel security processes. All GDS Personnel with access to production systems must have at least a security clearance of Security Checked (SC).</p>
```

Reply:
👍  - should have a lower case `p` though I think.

```suggestion
			<p>GDS follows government and Cabinet Office personnel security processes. All GDS personnel with access to production systems must have at least a security clearance of Security Checked (SC).</p>
```